### PR TITLE
chore: Remove invalid test files

### PR DESCRIPTION
**Pull request by lantoli**
## Proposed changes

_Jira ticket:_ [INTMDB-1201](https://jira.mongodb.org/browse/INTMDB-1201)

Remove invalid test files as they're not used in the contract tests.

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code
- [X] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

File: cfn-resources/auditing/test/inputs_1_invalid.json [hunk 1]
```diff
@@ -1,6 +1 @@
-{
-  "ProjectId": "",
-  "Profile": "default",
-  "AuditAuthorizationSuccess": "false",
-  "AuditFilter": 123124235
-}
+
```
Comment (AgustinBettati) [inline cfn-resources/auditing/test/inputs_1_invalid.json:1]: It appears the files are not fully deleted, has happened to me before

File: cfn-resources/auditing/test/cfn-test-create-inputs.sh [hunk 2]
```diff
@@ -61,9 +56,5 @@ jq --arg projectId "$projectId" \
 	'.ProjectId?|=$projectId |.Profile?|=$profile ' \
 	"$(dirname "$0")/inputs_2_update.json" >"inputs/inputs_2_update.json"
 
-jq --arg projectId "$projectId" \
-	--arg profile "$profile" \
-	'.ProjectId?|=$projectId |.Profile?|=$profile ' \
-	"$(dirname "$0")/inputs_2_invalid.json" >"inputs/inputs_2_invalid.json"
-
 ls -l inputs
+echo "mongocli iam projects delete ${projectId} --force"
```
[the remark this answers is not in the document]
Comment (lantoli) [reply cfn-resources/auditing/test/cfn-test-create-inputs.sh:60]: it's in more files, it's like a tip for you to delete the project. Resources are not automatically deleted for contract tests not in Github actions

File: cfn-resources/auditing/test/inputs_1_invalid.json (hunk 1 above)
Comment (lantoli) [reply cfn-resources/auditing/test/inputs_1_invalid.json:1]: wow, thanks, i don't how this happened. thank for noticing, I didn't event notice in this GH file compare page

Comment (andreaangiolillo) [reply cfn-resources/auditing/test/inputs_1_invalid.json:1]: there is a bug in the git hook we should fix it 😭

Comment (lantoli) [reply cfn-resources/auditing/test/inputs_1_invalid.json]: thanks, I'll take care of it

**Review by AgustinBettati — APPROVED** [review]

**Pull request merged**